### DISCUSSION TITLE
openAI() class is not showing after installation while trying to get topic repesentations

### DISCUSSION THREAD

**Discussion by AshokNagula** [MaartenGr/BERTopic#1045] [Q&A] [answered]
Hi @MaartenGr  , I am trying to use APIs for meaningful topic representations for my model. I have an API_KEY  from openAI but when i try to call the function representation = openAI()
its throwing an error like this 

ModuleNotFoundError                       Traceback (most recent call last)
/tmp/ipykernel_**1/30548**1.py in <cell line: 1>()
----> 1 representation_model = OpenAI()

~/anaconda3/envs/mytf1/lib/python3.10/site-packages/bertopic/_utils.py in __call__(self, *args, **kwargs)
     92 
     93     def __call__(self, *args, **kwargs):
---> 94         raise ModuleNotFoundError(self.msg)

ModuleNotFoundError: In order to use OpenAI you will need to install via;

`pip install openai`

although i installed openai , i am still getting the error .could you help me on this?

Note : Tried all other dependencies like python version, installation location seems everything is fine.

**Comment by MaartenGr** [comment, marked as answer]
It might be worthwhile to restart the notebook if you are working on that and then try to run the model. There might be something going on with the combination of jupyter and the kernel that you are currently using. Other than that, starting from a completely fresh environment and first installing everything there before running the code could be a solution.

**Reply by AshokNagula** [reply]
Thanks, its working but everytime i need to restart the kernal for new changes
